Client/mods/deathmatch/logic/CClientObjectManager.cpp

```diff
@@ -10,56 +10,6 @@
 
 #include "StdInc.h"
 
-// Generated by MTA10\utils\gentable\gentable_objmodels.cpp
```

Comment:
I think we can delete this file from the repo if we no longer use it anywhere else

Reply:
This file is used by serverside object manager.